reference-architecture/azure-ansible/store.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+RESOURCEGROUP=${1}
+USERNAME=${2}
+SSHPRIVATEDATA=${3}
+SSHPUBLICDATA=${4}
+SSHPUBLICDATA2=${5}
+SSHPUBLICDATA3=${6}
+
+export OSEUSERNAME=$2
+
+ps -ef | grep store.sh > cmdline.out
+
+mkdir -p /home/$USERNAME/.ssh
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3 >  /home/$USERNAME/.ssh/id_rsa.pub
+echo $SSHPRIVATEDATA | base64 --d > /home/$USERNAME/.ssh/id_rsa
+chown $USERNAME /home/$USERNAME/.ssh/id_rsa.pub
+chmod 600 /home/$USERNAME/.ssh/id_rsa.pub
+chown $USERNAME /home/$USERNAME/.ssh/id_rsa
+chmod 600 /home/$USERNAME/.ssh/id_rsa
+
+mkdir -p /root/.ssh
+echo $SSHPRIVATEDATA | base64 --d > /root/.ssh/id_rsa
+echo $SSHPUBLICDATA $SSHPUBLICDATA2 $SSHPUBLICDATA3   >  /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa.pub
+chmod 600 /root/.ssh/id_rsa.pub
+chown root /root/.ssh/id_rsa
+chmod 600 /root/.ssh/id_rsa
+
+
+yum -y update
+yum -y install targetcli
+yum -y install lvm2
+systemctl start target
+systemctl enable target
+systemctl restart target.service
+firewall-cmd --permanent --add-port=3260/tcp
+firewall-cmd --reload
+touch /root/.updateok
+pvcreate /dev/sdc /dev/sdd /dev/sde /dev/sdf /dev/sdg /dev/sdh /dev/sdi /dev/sdj
+parted --script -a optimal /dev/sdc mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdd mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sde mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdf mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdg mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdh mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdi mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+parted --script -a optimal /dev/sdj mklabel gpt mkpart primary ext2 1M 100% set 1 lvm on
+pvcreate /dev/sdc1 /dev/sdd1 /dev/sde1 /dev/sdf1 /dev/sdg1 /dev/sdh1 /dev/sdi1 /dev/sdj1
+vgcreate vg1 /dev/sdc1 /dev/sdd1 /dev/sde1 /dev/sdf1 /dev/sdg1 /dev/sdh1 /dev/sdi1 /dev/sdj1
+cat <<EOF | base64 --decode >  /root/ose_pvcreate_lun
```

Comment:
actually found the way of not escaping the world, its pretty easy - just didnt realize you colud do it that way. So now the whole script is clear-text, testing now.
actually time is more "time of day" on azure, ie if u hit it right, its really fast.
45 minutes

Also suspect it would be quicker if I cached the rpms.